zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -1822,6 +1823,43 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /ad-hoc-activities/activatable:
```

Comment:
`activatable` was one of the variants we discussed, but as this might not be a real term open to find a better one :)